src/app/main-page/container/toolbar-action-container/toolbar-action-container.component.ts

```diff
@@ -150,10 +153,20 @@ export class ToolbarActionContainerComponent implements OnInit, OnDestroy {
     }
 
     onOpenWarningsDialog() {
-        this.dialog.open(DialogImportWarningsComponent, {
-            // to do: add issues
-            data: "add issues here",
-        });
+        let fileName = "";
+
+        this.fileName$
+            .pipe(takeWhile(() => this.componentActive))
+            .subscribe((value) => {
+                if (value && value.trim().length > 0) {
+                    fileName = value;
+                }
+            });
```

Comment:
Is there a garantie that the subscription is called before you are using filename?